attestation-service/attestation-service/src/lib.rs

```diff
@@ -109,29 +71,50 @@ impl AttestationService {
             .map_err(|e| anyhow!("Cannot Set Policy: {:?}", e))
     }
 
+    fn accumulate_hash(materials: &[Vec<u8>]) -> Option<Vec<u8>> {
+        if materials.is_empty() {
+            return None;
+        }
+        let mut hasher = Sha384::new();
+        materials.iter().for_each(|m| hasher.update(m));
+        Some(hasher.finalize().to_vec())
+    }
+
     /// Evaluate Attestation Evidence.
     /// Issue an attestation results token which contain TCB status and TEE public key.
     pub async fn evaluate(&self, tee: Tee, nonce: &str, attestation: &str) -> Result<String> {
         let attestation = serde_json::from_str::<Attestation>(attestation)
             .context("Failed to deserialize Attestation")?;
-        let verifier = crate::verifier::to_verifier(&tee)?;
+        let verifier = verifier::to_verifier(&tee)?;
+
+        let report_data = Self::accumulate_hash(&[
+            nonce.as_bytes().to_vec(),
+            attestation.tee_pubkey.k_mod.as_bytes().to_vec(),
+            attestation.tee_pubkey.k_exp.as_bytes().to_vec(),
+        ]);
 
         let claims_from_tee_evidence = verifier
-            .evaluate(nonce.to_string(), &attestation)
+            .evaluate(
+                attestation.tee_evidence.as_bytes(),
+                report_data.as_deref(),
+                None,
```

Comment:
This field `init_data_hash` is currently hardcoded to `None`,  is this a temporary solution and expected to be updated later?  If yes, can you pls

- add some comments in this PR to explain what's this field expected to be? and how to set the value for it? 
- some comments to say this field is expected be updated in the later update. and better a github issue to track it.

Reply:
this change for this new field deserve a separate commit BTW, but it's up to you.

Reply:
Added. PTAL